data/duke.txt

```diff
@@ -0,0 +1,4 @@
+E | 1 | say hi | from 2 to 5
```

Comment:
Just a suggestion. You may want to consider adding duke.txt into your .gitignore file to avoid the constant file change when committing changes.